Dockerfile

```diff
@@ -51,13 +51,13 @@ RUN npm run build
 FROM scratch
 
 WORKDIR /prod
+COPY --from=server /tmp /tmp
```

Comment:
So. `scratch` doesn't have a `tmp` folder, and now our golang app needs the `tmp` directory in order to read the config file from `ioutil`. There was some chatter about this [here](https://www.reddit.com/r/docker/comments/8y2zyx/how_to_add_a_empty_directory_to_a_scratch_image/) (not particularly relevant), but it seems this is the way we need to go.